File: WebApplication1/Models/Documento.cs

namespace WebApplication1.Models
{
    public class Documento
    {
        public long id { get; set; }
        public string tipo { get; set; }
        public string estado { get; set; }
        public string fecha { get; set; }
        public string requisitos { get; set; }
        public Tramite tramite { get; set; }
        public string venFecha { get; set; }
        public string descripcion { get; set; }
        public string base64 { get; set; }
        public string areaDirigida { get; set; }

        public static implicit operator List<object>(Documento v)
        {
            throw new NotImplementedException();
        }
    }
}

File: src/main/java/com/example/api/models/Documento.java
package com.example.api.models;

import jakarta.persistence.*;
import lombok.Data;

import java.time.LocalDate;

@Entity
@Table(name = "documento")
@Data
public class Documento {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "documentoId")
    private Long id;

    @Column(name = "Tipo")
    private String tipo;

    @Column(name = "Estado")
    private String estado;

    @Column(name = "Fecha")
    private LocalDate fecha;

    @Column(name = "Requisitos")
    private String requisitos;

    @ManyToOne
    @JoinColumn(name = "tramiteid")
    private Tramite tramite;

    @Column(name = "venFecha")
    private LocalDate venFecha;

    private String descripcion;

    @Lob
    private byte[] adjunto;

    @Transient
    private String base64;

    @Column(name = "area_dirigida")
    private String areaDirigida;

}
